data/pigui/modules/saveloadgame.lua

```diff
@@ -243,6 +258,7 @@ end
 --=============================================================================
 
 function SaveLoadWindow:makeFilteredList()
+	local profileEndScope = utils.profile("SaveLoadWindow::makeFilteredList()")
```

Comment:
Leftover profiling call here.

Reply:
The change is impressive, even Debug builds now load the list of saves almost instantly, down from about 14 seconds, on my PC with about 40 saves.

I'll squash/merge/etc and mark the PR as ready then. Thank you for all of your input!